packages/wrangler/src/config-path.ts

```diff
@@ -0,0 +1,25 @@
+import fs from "node:fs";
+import os from "node:os";
+import path from "node:path";
+import xdgAppPaths from "xdg-app-paths";
+
+export function getConfigPath() {
+	//TODO: We should implement a custom path --global-config
+	const configDir = xdgAppPaths(".wrangler").config(); // New XDG compliant config path
+	const legacyConfigDir = path.join(os.homedir(), ".wrangler"); // Legacy config in user's home directory
+
+	function isDirectory(configPath: string) {
+		try {
+			return fs.statSync(configPath).isDirectory();
+		} catch (error) {
+			// ignore error
+			return false;
+		}
+	}
```

Comment:
NIT: this function doesn't need to live inside `getConfigPath()` let's move it to the top level of the file. It is just adding noise here.